crates/context/src/lib.rs

```diff
@@ -73,7 +73,7 @@ impl ContextManager {
         network_client: NetworkClient,
     ) -> EyreResult<Self> {
         let client_config = config.client.clone();
-        let config_client = ContextConfigClient::from_config(&client_config);
+        let config_client = Client::from_config(&client_config);
```

Comment:
alias this to `ExternalClient` or something like that

from my reorganization comment, this should later be in a module called

`calimero_external::Client`